exercises/concept/calculator-conundrum/src/test/java/CalculatorConundrumTest.java

```diff
@@ -53,16 +60,30 @@ public void throwExceptionForUnknownOperation() {
     }
 
     @Test
+    @Tag("task:2")
+    @DisplayName("The calculate method throws an IllegalOperationException with custom message when passing null as operation")
```

Comment:
```suggestion
    @DisplayName("The calculate method throws an IllegalArgumentException with custom message when passing null as operation")
```

Reply:
thanks for the review, I just updated with a fixing commit